src/CoolTensors.jl

```diff
@@ -139,4 +140,13 @@ function Base.typed_hvcat(t::Tensor{<:Any,N,ipos}, alt::Tuple, i...) where {N,ip
     t[I]
 end
 
+using TensorCore
+export tensor, ⊗
```

Comment:
I wonder, whether we want to export `tensor` as well, since that might be a bit confusing when the type is called `Tensor`. What do you think about exporting an alias `tensor_product` instead?

Reply:
I made it just `⊗` for now.